### DISCUSSION TITLE
v4.3.0-RC1 not running on Windows

### DISCUSSION THREAD

**Discussion by jens-auer** [containers/podman#16100] [General]
Hi,

I tried the [v4.3.0-RC1](https://github.com/containers/podman/releases/tag/v4.3.0-rc1) prerelease on my Windows machine but only get an error:

```
podman
time="2022-10-10T09:23:48+02:00" level=error msg="this function is not implemented for windows"
```

I was running 4.2.1 before and uninstalled it, so I think that the machine is configured correctly. WSL is installed:

```
wsl --status
Default Distribution: openSUSE-Tumbleweed
Default Version: 2

Windows Subsystem for Linux was last updated on 6/28/2022
WSL automatic updates are on.

Kernel version: 5.10.102.1
```

Since it is prerelease it is not a big issue for me, but I just wanted to report it.

**Comment by jens-auer** [comment]
I did not create an issue because it is a pre-release version.

**Reply by rhatdan** [reply]
This would be a big issue for us, thanks for reporting it.

**Comment by mheon** [comment]
I believe this was fixed already by @n1hility

**Comment by n1hility** [comment]
That's correct this was fixed in containers/common#1169, which will be included in the next release.
